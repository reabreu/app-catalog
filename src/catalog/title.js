import React from "react";
import { Header1 } from "./styles";

export const Title = () => <Header1>App Catalog</Header1>;
