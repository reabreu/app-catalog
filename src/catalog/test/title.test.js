import React from "react";
import { render } from "@testing-library/react";
import { Title } from "../title";

describe("<Title /> spec", () => {
  it("Renders Title", () => {
    const { asFragment } = render(<Title />);

    expect(asFragment()).toMatchSnapshot();
  });
});
