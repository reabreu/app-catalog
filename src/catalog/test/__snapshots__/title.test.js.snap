// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> spec Renders Title 1`] = `
<DocumentFragment>
  <h1
    class="sc-AxjAm jWlVxK"
  >
    App Catalog
  </h1>
</DocumentFragment>
`;
